From 1b69941dcdf1f4f3762f9dae0d239d9bead9b638 Mon Sep 17 00:00:00 2001
From: Josse Colpaert <jco@odoo.com>
Date: Fri, 1 Feb 2019 18:33:50 +0100
Subject: [PATCH] [FIX] check locations

---
 addons/stock/models/stock_picking.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/addons/stock/models/stock_picking.py b/addons/stock/models/stock_picking.py
index 3e3e99c..ab1bd9e 100644
--- a/addons/stock/models/stock_picking.py
+++ b/addons/stock/models/stock_picking.py
@@ -737,6 +737,7 @@ class Picking(models.Model):
                         else:
                             flag = not quant.package_id.id
                         flag = flag and (ops.owner_id.id == quant.owner_id.id)
+                        flag = flag and (ops.location_id.id == quant.location_id.id)
                         if flag:
                             if not lot_qty:
                                 max_qty_on_link = min(quant.qty, qty_to_assign)
-- 
2.7.4

